ddtrace/internal/wrapping.py

```diff
@@ -346,6 +346,11 @@ def wrap(f, wrapper):
         f.__defaults__,
         f.__closure__,
     )
+    try:
+        cast(WrappedFunction, wrapped).__dd_wrapped__ = cast(FunctionType, cast(WrappedFunction, f).__dd_wrapped__)
```

Comment:
This is quite hard to read with all the casting, could we maybe cast on a separate line first?